beets/library.py

```diff
@@ -186,6 +186,7 @@ def parse(self, key):
         for flat, sharp in self.ENHARMONIC.items():
             key = re.sub(flat, sharp, key)
         key = re.sub(r'[\W\s]+minor', 'm', key)
+        key = re.sub(r'[\W\s]+major', '', key)
```

Comment:
Previously, the word minor was shortened to 'm' to follow naming conventions. While writing this plugin, I realized that AcousticBrainz returns the key_scale with either "minor" or "major." Since "major" is implied when "minor" is not present in some form, I redacted major.